routes/v1.js

```diff
@@ -1311,9 +1362,13 @@ api.declare({
 
   // If run isn't running we had a conflict
   if (task.runs.length - 1 !== runId || run.state !== 'running') {
-    return res.status(409).json({
-      message: "Run is resolved, or not running"
-    });
+    return res.reportError(
+      "RequestConflict",
+      "Run {{runId}} on task {{taskId}} is resolved or not running.",
+      {
+        taskId,
+        runId: runId.toString()
```

Comment:
I've proposed: https://github.com/taskcluster/taskcluster-lib-api/pull/12

So we don't have to do `toString()` and ` task.deadline.toJSON()` :)
Let me know if you like it.